### ISSUE TITLE
[NODE] `ADMIN_CREDENTIALS_FILE` causes chainlink to fail

### ISSUE THREAD

**Issue by yorickdowne** [smartcontractkit/chainlink#4128]
**Description**
If `ADMIN_CREDENTIALS_FILE` is set, chainlink fails with:

```
{"level":"fatal","ts":1616895577.6483572,"caller":"logger/default.go:165","msg":"Error loading API credentials","error":null,"credentialsFile":"/secrets/apicredentials.txt","stacktrace":"github.com/smartcontractkit/chainlink/core/logger.Fatalw\n\t/chainlink/core/logger/default.go:165\nmain.NewProductionClient\n\t/chainlink/core/main.go:35\nmain.main\n\t/chainlink/core/main.go:14\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:204"}
2021/03/28 01:39:37 Command exited with error: exit status 1

```

Specifying the same file via `--api` works.

**Basic Information**

- Operating System: Ubuntu 20.04, docker
- Commit: Chainlink Node 0.10.3 at commit 5dbbc21f13fa00ee724c439273156834c45ab93e

**Environment Variables**
`ADMIN_CREDENTIALS_FILE=/secrets/apicredentials.txt`

**Steps to Reproduce**

- Create `apicredentials.txt` file and make sure it is host-volume mounted to `/secrets/apicredentials.txt` when starting chainlink
- Set env variable `ADMIN_CREDENTIALS_FILE=/secrets/apicredentials.txt`
- Start chainlink with entrypoint `chainlink node start --password /secrets/wallet-password.txt` or `chainlink node start --password /secrets/wallet-password.txt --api /secrets/apicredentials.txt`
- Observe fatal error and exit

- Comment out env variable `ADMIN_CREDENTIALS_FILE=/secrets/apicredentials.txt`
- Start chainlink with entrypoint `chainlink node start --password /secrets/wallet-password.txt` or `chainlink node start --password /secrets/wallet-password.txt --api /secrets/apicredentials.txt`
- Observe correct behavior

**Comment by PatrickAlphaC** [comment]
Thanks for this request! Asking to get some other eyes on this.

**Comment by j16r** [comment]
@yorickdowne I believe this issue is fixed in 0.10.4

**Comment by yorickdowne** [comment]
I'll test and see! Thank you!

**Comment by yorickdowne** [comment]
Confirmed resolved in 0.10.4
